client/src/ui/utils/utils.tsx

```
@@ -59,6 +59,14 @@ export function divideByPrecision(value: number): number {
   return value / EternumGlobalConfig.resources.resourcePrecision;
 }
 
+export function roundDownToPrecision(value: bigint, precision: Number) {
```

Biome

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)